packages/next-app/src/wagmi/setPoolMetadata.ts

```diff
@@ -1,25 +0,0 @@
-import { Address } from "@bleu-balancer-tools/shared";
```

Comment:
I gather this was simply not being used anymore?

Reply:
yes, we are using the wagmi generated hook for a while now. wrote it on the commit message https://github.com/bleu-studio/balancer-tools/pull/157/commits/16ffa083fb5f65ad84811f67fb213b19d1eb7e51